docs/Developers/Git_Basics.rst

```diff
@@ -453,3 +453,7 @@ use an interactive rebasing session. This opens an editor where you can enter
 commands (described below) for each commit to be rebased. These commands
 determine how individual commits will be transferred to the new base. You can
 also reorder the commit listing to change the order of the commits themselves.
+
+If you would like more information/commands, please use your favourite search
+engine to look for it. You could also read the
```

Comment:
You could also read this "git documentation" to learn more.
link the quoted part.